AzureVMNoiseDataset2024.md

```diff
@@ -0,0 +1,121 @@
+We used [SSDv2 Disks](https://learn.microsoft.com/en-us/azure/virtual-machines/disks-types#premium-ssd-v2) as the "remote disk", and a [Premium SSDs](https://learn.microsoft.com/en-us/azure/virtual-machines/disks-types#premium-ssds) as the "local disk" in the tests.
+This dataset is the data that is described in, and analyzed in the EuroSys 2025 paper [TUNA: Tuning Unstable and Noisy Cloud Applications](https://www.microsoft.com/en-us/research/publication/tuna-tuning-unstable-and-noisy-cloud-applications/).
+A set of benchmarks were used to attempt to cover the main components in the VM, with the exception of the network: Cache, CPU, Disk, Memory, OS.
+Additionally, two end to end applications were also benchmarked: PostgreSQL and Redis.
+
+### Main characteristics
+
+* Dataset size: 277 MB
+* Number of files: 1520 files
+* Duration: 483 days
+* Average Benchmark Suite Duration: 130 minutes
+* Long Lived VM Characteristics:
+  * Metrics Collected: 3,661,602
+  * Total VMs: 12
+* Short Lived VM Characteristics:
+  * Metrics Collected: 3,375,618
+  * Total VMs: 43617
+  * Time between VM instantiations: 40 minutes
+
+## Using the Data
+
+### License
+
+The data is made available and licensed under a [CC-BY Attribution License](https://github.com/Azure/AzurePublicDataset/blob/master/LICENSE). By downloading it or using them, you agree to the terms of this license.
+
+### Attribution
+
+If you use this data for a publication or project, please cite the accompanying paper:
+
+> Johannes Freischuetz, Konstantinos Kanellis, Brian Kroth, and Shivaram Venkataraman. 2025. [TUNA: Tuning Unstable and Noisy Cloud Applications](https://www.microsoft.com/en-us/research/publication/tuna-tuning-unstable-and-noisy-cloud-applications/). In Proceedings of the Twentieth European Conference on Computer Systems (EuroSys '25). Association for Computing Machinery, New York, NY, USA
+
+> @inproceedings {TUNA, author = {Johannes Freischuetz and Konstantinos Kanellis and Brian Kroth, and Shivaram Venkataraman}, title = {TUNA: Tuning Unstable and Noisy Cloud Applications}, booktitle = {EuroSys '25: Proceedings of the Nineteenth European Conference on Computer Systems}, publisher = {Association for Computing Machinery}, address = {New York, NY, USA}, year={2025}, month = {mar}}
+
+We provide the traces as they are, but are willing to help researchers understand and use them. So, please let us know of any issues or questions by sending email to our  [mailing list](mailto:azurepublicdataset@service.microsoft.com).
+
+### Downloading
+
+You can download the dataset here:  [**vm-noise-data**](https://github.com/Azure/AzurePublicDataset/raw/master/data/vm-noise-data).
+
+## Schema and Description
+
+### Layout
+
+* 1 file per measurement unit, partitioned using Hive style table partitioning layout:
+
+`test_suite=<test_suite>/test_name=<test_name>/vm_lifespan=<short|long>/vm_region=(eastus|westus2)/vm_sku=(B8ms|D8s_v5)/unit=<unit>.csv`
```

Comment:
```suggestion
`test_suite=<test_suite>/test_name=<test_name>/vm_lifespan=(short|long)/vm_region=(eastus|westus2)/vm_sku=(B8ms|D8s_v5)/unit=<unit>.csv`
```

formatting consistency fixup

Reply:
Would also be good to have an `example.py` that shows how to read the contents into a data frame.